README.md

```diff
@@ -1,8 +1,25 @@
 # Kubernetes Python Client
 
 [![Build Status](https://travis-ci.org/kubernetes-incubator/client-python.svg?branch=master)](https://travis-ci.org/kubernetes-incubator/client-python)
+[![PyPI version](https://badge.fury.io/py/kubernetes.svg)](https://badge.fury.io/py/kubernetes)
 
-Python clients for talking to a [kubernetes](http://kubernetes.io/) cluster.
+Python client for the [kubernetes](http://kubernetes.io/) API.
+
+## Installation
+
+From source:
+
+```
+git clone https://github.com/kubernetes-incubator/client-python.git
+cd client-python
+pip install .
+```
+
+From [PyPi](https://pypi.python.org/pypi/kubernetes/1.0.0-alpha.2) directly:
```

Comment:
link main pypi kubernetes page instead of an specific version? e.g. https://pypi.python.org/pypi/kubernetes/